docs/eleccion_runtime.md

```diff
@@ -0,0 +1,21 @@
+# Elección de runtime
+
+La elección del runtime se hará en base a los criterios establecidos en el archivo de 
+[criterios de elección](./criterios_eleccion_herramientas.md).
+
+## Opciones disponibles
+
+### Node.js
+Es la opción mas popular y permite la ejecución asíncrona de javascript. No establece grandes criterios de seguridad y para ejecutar TypeScript debemos compilar previamente el código TypeScript a JavaScript. Es gratuito y open source. Importa modulos mediante CommonJS, lo que obliga a descargar/instalar dichos módulos antes de usarlos. 
+
+### Deno
+Deno surgió como una mejora más segura para Node.js. Fue creado por la misma persona que Node.js y cuenta con soporte nativo para TypeScript sin necesidad de compilar a JavaScript. Cuenta con herramientas propias como lintern y test runner. Además, también mejora la velocidad de ejecución frente a Node.js. Al igual que Node.js, es gratuito y de código abierto. En cuanto a los módulo, se importan median ECMAScript, lo que significa que no hay que descargarlos y pueden ser importados mediante URLs. 
```

Comment:
porfa, en general, tratad sólo de poner la información que sea pertinente a la elección que hayáis hecho.

Reply:
resuelto en: https://github.com/raultl12/TeamFinder/pull/21/commits/eae46f18d42c0db750e94e1bc274c6b507d91efa